content/docs/capabilities/getting-users-identity.md

```diff
@@ -167,5 +167,5 @@ In an actual client, you'll want to ensure that all the other claims values are
 [key management service]: https://en.wikipedia.org/wiki/Key_management
 [nist p-256]: https://csrc.nist.gov/csrc/media/events/workshop-on-elliptic-curve-cryptography-standards/documents/papers/session6-adalier-mehmet.pdf
 [pass identity headers]: /docs/reference/routes/pass-identity-headers-per-route
-[signing key]: /docs/reference/signing-key
-[signing key file]: /docs/reference/signing-key-file
+[signing key]: /docs/reference/signing-key#signing-key
+[signing key file]: /docs/reference/signing-key#signing-key-file
```

Comment:
I think these can both link to the same place now that these are combined into one.
```suggestion
[signing key]: /docs/reference/signing-key
[signing key file]: /docs/reference/signing-key
```

Or better yet, maybe line 101 can be simplified to just

```
In order to use the `/.well-known/pomerium/jwks.json` endpoint you must set the [Signing Key] configuration option.
```

and then we can delete line 171 here.

Reply:
Done.